ImageSource/Local/LocalImageSource.swift

```diff
@@ -5,11 +5,13 @@ import MobileCoreServices
 public final class LocalImageSource: ImageSource {
     
     public let path: String
+    public let url: URL?
     
     // MARK: - Init
     
-    public init(path: String, previewImage: CGImage? = nil) {
+    public init(path: String, url: URL? = nil, previewImage: CGImage? = nil) {
```

Comment:
There should be a separate initializer for that. This initializer accepts pass both path and url, which is confusing.